server/server.js

```diff
@@ -0,0 +1,57 @@
+import express from "express";
+import axios from "axios";
+import dotenv from "dotenv";
+import cors from "cors";
+
+dotenv.config();
+
+const app = express();
+const PORT = 8080;
+
+app.use(
+  cors({
+    origin: process.env.URL || "http://localhost:5173",
+    allowedHeaders: "Origin, X-Requested-With, Content-Type, Accept",
+  })
+);
```

Comment:
CORS are security rules that browsers use to control how they make requests to APIs and other sources of data. When a browser tries to make requests to a different domain, the requestee has to explicitly allow requests from the requester. The requester's server has to have a specific CORS header, showing that it has access to request data from the requestee. If the header isn't right the request is blocked. Overall this is supposed to prevent malicious or illegitimate requests.